indico/modules/events/surveys/models/surveys.py

```diff
@@ -135,7 +168,7 @@ def event(self, event):
     def has_ended(self):
         return bool(self.end_dt) and self.end_dt <= now_utc()
 
-    @property
+    @hybrid_property
     def has_started(self):
         return bool(self.start_dt) and self.start_dt <= now_utc()
```

Comment:
like i thought, this simply evaluates to `start_dt <= now` when SQL is generated, i.e. the `is not null` check is missing. I believe SQLAlchemy simply executes the method with `self` being set to the class itself, so it's `True and ...` so in the end only the `...` is returned. anyway, fixed in the commit i mentioned above.